src/commands/init.ts

```diff
@@ -17,21 +20,60 @@ export default class Init extends Command {
   ]
 
   static flags = {
-    version: Flags.string({ description: 'project version' })
+    version: Flags.string({ description: 'Project version' }),
+    'from-hub': Flags.boolean({ description: 'Clones a bundle from Entando Hub' }),
+    'hub-url': Flags.string({ description: 'Custom Entando Hub url' }),
   }
 
   public async run(): Promise<void> {
     const { args, flags } = await this.parse(Init)
 
-    const initializer = new InitializerService({
-      parentDirectory: process.cwd(),
-      name: args.name as string,
-      version: flags.version ?? DEFAULT_VERSION
-    })
+    const parentDirectory = process.cwd()
+    const name = args.name as string
+    const version = flags.version ?? DEFAULT_VERSION
 
-    // Displaying spinner while performing the scaffolding
-    CliUx.ux.action.start(`Initializing a new bundle project ${args.name}`)
-    await initializer.performScaffolding()
-    CliUx.ux.action.stop()
+    const options = { parentDirectory, name, version }
+
+    const initializer = new InitializerService(options)
+
+    if (flags['from-hub']) {
+      CliUx.ux.action.start(`Initializing hub service for your new bundle project ${args.name}`)
+      const hubService = new HubService(flags['hub-url'])
+      const bundleGroups = await hubService.loadBundleGroups()
+      CliUx.ux.action.stop()
+      const selectedBundleGroup = await this.promptSelectBundleGroup(bundleGroups)
+      const bundles: Bundle[] = await hubService.loadBundlesFromBundleGroup(selectedBundleGroup)
+      const selectedBundle: Bundle = await this.promptSelectBundle(bundles, selectedBundleGroup)
+
+      CliUx.ux.action.start(`Gathering selected bundle ${selectedBundle.bundleName} to new project ${args.name}`)
+      await initializer.performCloneBundle(selectedBundle)
```

Comment:
Just pass the git repo as we don't use other params

Reply:
git repo now being passed instead of `Bundle`